### PR TITLE
feat(artifacts): bitbucket, gitlab artifact account setup

### PR DESCRIPTION
**Pull request by ghost**
should I update the reference/halyard/commands.md file too or does that update from the halyard/docs at some point? @lwander

### CODE REVIEW

**Review by lwander — APPROVED** [review]
LGTM -- will wait on @dorbin 

The commands page is automatically updated whenever we publish a halyard release.

File: setup/artifacts/gitlab.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: single
+title:  "Configure a Gitlab artifact account"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+These steps show you how to configure a Gitlab artifact account so that
```
Comment (dorbin) [inline setup/artifacts/gitlab.md]: s/Gitlab/GitLab/
(throughout)

Comment (dorbin) [reply setup/artifacts/gitlab.md]: (except for in commands and urls, obviously ;)

File: setup/artifacts/s3.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+title:  "Configure S3 Artifact"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that read data from artifacts can read S3 files directly.
+
+Several settings can be configured.
```
Comment (dorbin) [inline setup/artifacts/s3.md]: I think you can delete this sentence, unless you're going to enumerate the several setting here.

**Review by dorbin — CHANGES_REQUESTED** [review]
Can you also add these to the [nav tree](https://github.com/spinnaker/spinnaker.github.io/blob/master/_data/navigation.yml)?

**Comment by ghost** [comment]
done

File: _data/navigation.yml [hunk 3]
```diff
@@ -101,6 +101,12 @@ setup:
             url: /setup/artifacts/github/
           - title: HTTP
             url: /setup/artifacts/http/
+          - title: Bitbucket
+            url: /setup/artifacts/bitbucket/
+          - title: GitLab
+            url: /setup/artifacts/gitlab/
+          - title: S3
+            url: /setup/artifacts/http/
```
Comment (dorbin) [inline _data/navigation.yml]: :+1:

**Review by dorbin — APPROVED** [review]

**Comment by lwander** [comment]
Looks like we need to publish halyard to satisfy the linter... https://travis-ci.org/spinnaker/spinnaker.github.io/builds/387940814#L2718

Let's let the nightly tests pass & then publish tomorrow

File: _data/navigation.yml (hunk 3 above)
Comment (ghost) [reply _data/navigation.yml]: lol

**Pull request merged**